templates/settings/payment.php

```diff
@@ -10,44 +10,113 @@
 
 do_action( 'dokan_payment_settings_before_form', $current_user, $profile_info ); ?>
 
-<form method="post" id="payment-form"  action="" class="dokan-form-horizontal">
+<div class="dokan-payment-settings-summary">
+    <h2 id="vendor-dashboard-payment-settings-error"></h2>
+    <div class="payment-methods-listing-header">
+        <h2> <?php esc_html_e( 'Payment Methods', 'dokan-lite' ); ?></h2>
+        <div id="vendor-dashboard-payment-settings-toggle-dropdown">
+            <button id="toggle-vendor-payment-method-drop-down" class="dokan-btn dokan-btn-success"> <?php esc_html_e( 'Add Payment Method', 'dokan-lite' ); ?></button>
+            <div id="vendor-payment-method-drop-down-wrapper">
+                <div id="vendor-payment-method-drop-down">
+                    <?php
+                    $has_methods_in_dropdown = false;
+                    $icon_mismatch_map       = [
+                        'dokan-stripe-connect'     => 'stripe',
+                        'dokan-paypal-marketplace' => 'paypal'
+                    ];
+                    $mis_match_methods       = [ 'stripe', 'moip' ];
+                    $unused_methods          = get_unused_payment_methods( $methods, $profile_info['payment'], $mis_match_methods );
```

Comment:
Never name a global function without prefix.
make it `dokan_get_unused_payment_methods`